src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/RulesDownloadService.swift

```diff
@@ -24,11 +24,16 @@ class RulesDownloadService: RulesDownloadServiceProviding {
 	}
 	
 	// MARK: - Protocol RulesDownloadServiceProviding
-	
+
 	func downloadRules(
 		ruleType: HealthCertificateValidationRuleType,
 		completion: @escaping (Result<[Rule], DCCDownloadRulesError>) -> Void
 	) {
+		guard ruleType != .boosterNotification else {
+			Log.error("invalid use for booster rules download")
+			fatalError("invalid use for booster rules download")
+		}
+
```

Comment:
Can we just remove the enum case `.boosterNotification`?

Reply:
it's not that simple because of the error handling of all DCCRules